lib/taste_tester/host.rb

```diff
@@ -40,7 +40,8 @@ def initialize(name, server)
 
     def runchef
       logger.warn("Running '#{TasteTester::Config.command}' on #{@name}")
```

Comment:
I know this isn't a thing you are changing, but wtf is `TasteTester::Config.command` ? I don't see it in config.rb.... what are we telling people here?